gtmn1kqn.jrj.txt

```diff
@@ -0,0 +1,63 @@
+﻿Merge remote-tracking branch 'refs/remotes/origin/master' into linking-LCA-database-to-optimization-calculations
```

Comment:
what is this file? i'm not sure you wanted to merge this.

Reply:
yes, you are right. I did not intend to add this, but merging master has created this